docs/getting_started.rst

```diff
@@ -0,0 +1,78 @@
+===============
+Getting Started
+===============
+-------------------
+What is Wifiphisher
+-------------------
+Wifiphisher  is a security tool that mounts automated victim-customized phishing attacks against
+WiFi clients in order to obtain credentials or infect the victims with malwares. It is primarily
+a social engineering attack that unlike other methods it does not include any brute forcing. It
+is an easy way for obtaining credentials from captive portals and third party login pages
+(e.g. in social networks) or WPA/WPA2 passwords.
+
+----------------------
+How to get Wifiphisher
+----------------------
+There are three ways to obtain Wifiphisher
+
+^^^^^^^^^^^^
+Using Pip
+^^^^^^^^^^^^
+
+This method is the simplest and easiest way to obtain Wifiphisher. All you have to do is Simply
+run the following command in the terminal
+
+.. code:: bash
+
+ [sudo] pip install --upgrade wifiphisher
+
+.. image:: _static/getting_started/install_pip.gif
+
+.. note::
+  Using this method is recommended
+
+.. warning::
+  ``pip`` must be installed on your system.
+
+.. seealso::
+  pip_ and ``pip --help`` for more information.
+
+.. _pip: https://pypi.python.org/pypi/pip/
+
+^^^^^^^^^^^^
+Using git
+^^^^^^^^^^^^
+This method involves using the ``git clone`` command. Simply run the following command from the
+terminal
+
+.. code:: bash
+
+  git clone https://github.com/wifiphisher/wifiphisher.git
+
+.. image:: _static/getting_started/install_git.gif
+
+.. warning::
+  ``git`` must be installed on your system.
+
+.. seealso::
+  Git_ and ``git clone --help`` for more information.
+
+.. _Git: https://git-scm.com/
+
+^^^^^^^^^^^^^^^^^^^^^^^
+Using browser
+^^^^^^^^^^^^^^^^^^^^^^^
+This method involves downloading all the files using a browser.
+
+  #. Visit the projects page on Github_ |page|
+  #. Press the ``clone or download`` button |download|
+  #. Click the ``Download ZIP`` button |zip|
+
+  .. |page| image:: _static/getting_started/install_browser_page.gif
+    :scale: 60 %
+  .. |download| image:: _static/getting_started/install_browser_download.gif
+    :scale: 60 %
+  .. |zip| image:: _static/getting_started/install_browser_zip.gif
+    :scale: 50 %
+
+.. _Github: https://github.com/wifiphisher/wifiphisher
```

Comment:
Looking through the docs in there I didn't see anything that resembles that. To give you a better idea when fully made to HTML it would look like this:

![screenshot_20170317_141626](https://cloud.githubusercontent.com/assets/25112506/24057082/7e2dea72-0b1c-11e7-9f12-319f70dbf416.png)